libunwind/src/UnwindRegistersSave.S

```diff
@@ -842,7 +842,7 @@ DEFINE_LIBUNWIND_FUNCTION(__unw_getcontext)
   stp    c28,c29, [c0, #0x1c0]
   str    c30,     [c0, #0x1e0]
   mov    c1,csp
-  str    c1,      [c0, #0x1F0]
+  str    c1,      [c0, #0x1f0]
```

Comment:
Local style is also uppercase....

Reply:
Yes, but this is more consistent with the rest of the pure cap blocks here which are all lowercase.